### PR TITLE
User model

### PR DESCRIPTION
**Pull request by shernanm**
## Summary

* The migration that allows the creation of the users table in the database was created.
* The model that was used in the application was defined.

## Trello Card

https://trello.com/c/7nWothjC

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        unique: true,
+        allowNull: false,
+        type: DataTypes.STRING
+      },
+      password: {
+        allowNull: false,
+        type: DataTypes.STRING
+      }
+    },
+    {}
```
Comment (molinajulian) [inline app/models/user.js]: Check `underscore` param here for transform camelCase to snake_case

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      firstName: {
+        allowNull: false,
+        type: DataTypes.STRING,
+        field: 'first_name'
```
Comment (molinajulian) [inline app/models/user.js]: Delete all key `field` and use `underscore`

File: migrations/migrations/20190529172535-create-user.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('Users', {
+      id: {
```
Comment (molinajulian) [inline migrations/migrations/20190529172535-create-user.js:5]: Delete all keys `field` and create all properties with key in snake_case

**Review by molinajulian — CHANGES_REQUESTED** [review]

File: migrations/migrations/20190529172535-create-user.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('Users', {
```
Comment (raparicio6) [inline migrations/migrations/20190529172535-create-user.js]: tables names in database are in lower case

**Review by raparicio6 — CHANGES_REQUESTED** [review]

File: migrations/migrations/20190529172535-create-user.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      created_at: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updated_at: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      deleted_at: {
+        allowNull: false,
```
Comment (molinajulian) [inline migrations/migrations/20190529172535-create-user.js]: This can be null

**Review by molinajulian — APPROVED** [review]

**Review by raparicio6 — APPROVED** [review]

**Pull request merged**